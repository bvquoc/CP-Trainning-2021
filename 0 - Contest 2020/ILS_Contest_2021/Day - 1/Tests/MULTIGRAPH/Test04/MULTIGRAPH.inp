199 199
758834 473129 356416 872109 363267 740102 737475 88539 3744 379494 100927 230499 340052 442496 448452 443805 326515 761358 603107 5128 348686 765989 333789 84303 335250 779726 491193 347770 566938 447678 65245 2210 690090 220942 838653 117786 817554 360986 817500 484208 454237 178806 808288 375762 166674 422040 115649 184079 740264 692886 125839 182316 646457 226563 862347 670570 101171 623595 453463 923635 601790 557687 535865 546769 198379 58384 643293 238814 303552 969856 153820 887401 681072 903794 893869 785731 88432 824836 292456 859176 757002 459390 104442 498520 766346 784821 944292 865263 691919 332883 497399 696300 709074 160581 13272 873581 400219 732125 611533 253503 685408 1995 45419 493934 99708 800310 667442 347300 280485 92237 265037 46654 891152 453952 120285 146026 194604 107562 703326 491558 316077 908231 432681 377052 678601 693099 135812 520772 833601 373843 647562 750551 494800 677975 466847 772763 763992 655350 718908 216953 936608 149541 249890 429992 353025 194911 126527 452129 889974 335928 757629 2945 610958 191191 103421 724365 832823 156315 43738 75446 927612 517584 761319 35082 138429 383001 34928 752952 175878 670008 513501 798697 290140 733037 609489 51107 456 25897 316277 856494 756952 900105 416345 242806 289196 693276 420397 348602 712838 362552 892434 444866 879287 60770 862988 174517 215582 779344 271016
2 30 26
1 177 590598
2 41 53
1 102 929705
1 32 260183
1 152 813491
1 9 370420
1 20 932784
1 95 324700
2 76 127
1 178 71493
2 55 183
1 167 749021
2 122 115
1 164 552375
2 59 163
1 159 706462
2 42 119
2 124 68
1 103 236708
1 112 833774
1 176 419372
1 66 830797
2 131 48
2 167 149
1 194 310268
2 62 101
1 31 964277
1 178 333669
1 160 272274
1 24 692944
1 77 978441
1 8 424000
1 110 312476
2 15 20
1 105 761970
1 11 510250
1 57 208939
1 155 498881
1 83 711292
2 168 102
1 118 126133
1 47 802723
1 36 897731
1 115 319375
1 51 385741
1 118 641892
1 147 643488
2 77 194
1 127 150065
2 61 194
1 165 429884
2 122 74
2 106 129
2 159 82
1 116 984855
1 142 848527
1 127 495828
1 71 196237
1 158 331879
2 80 44
1 94 894232
1 45 477645
1 196 636728
1 169 536376
2 135 188
2 97 34
1 42 535860
1 52 296174
2 50 146
2 117 189
2 184 130
1 48 329179
1 154 966084
1 117 565506
2 126 75
1 146 506148
1 71 336660
1 65 385088
1 57 902916
1 197 222300
1 140 909931
1 34 607425
1 197 793868
2 189 25
1 54 525618
1 12 854115
1 103 334829
1 68 554808
2 6 56
1 184 853687
2 64 73
1 143 876402
1 100 330507
2 87 76
1 32 797374
1 111 667557
2 146 169
1 199 850331
1 160 816690
1 109 916258
2 163 156
1 185 543898
1 173 625820
1 79 532392
1 52 429832
2 54 69
1 69 343327
1 194 743948
1 110 394583
1 121 809888
1 179 740783
2 167 35
1 115 658406
1 95 388185
1 17 814769
2 73 32
1 48 560280
1 100 343894
1 158 372291
2 116 111
1 90 848376
2 107 55
2 54 162
1 178 442569
1 23 901348
1 103 658863
1 25 834199
1 150 736454
1 71 661786
1 13 372538
1 69 541604
1 100 969266
1 108 418804
1 28 775437
1 188 797694
1 21 813328
1 145 527960
1 3 570198
1 38 473231
1 190 446002
2 109 183
1 5 770556
1 9 383283
1 158 951508
1 13 919505
2 138 102
1 130 392948
1 44 645344
1 124 731074
1 10 735989
2 180 26
1 166 412968
1 9 485984
2 90 148
1 65 896671
2 71 179
1 51 841624
1 95 632346
2 119 86
1 130 718480
2 83 91
1 110 651035
1 97 977455
2 189 74
1 166 679490
1 183 468146
1 108 870373
1 176 600172
1 187 623494
2 5 62
1 46 857157
1 8 894010
1 52 652252
1 165 657232
1 144 857265
1 123 687788
1 14 721780
1 178 583068
1 16 656116
1 192 878484
1 190 490729
1 30 792381
1 15 763820
2 114 144
1 148 662297
1 59 511416
2 56 103
1 114 496051
2 181 58
1 41 705776
1 82 941523
2 50 189
1 135 616348
1 183 603694
1 2 546694
2 154 135
1 38 718168
2 38 88
