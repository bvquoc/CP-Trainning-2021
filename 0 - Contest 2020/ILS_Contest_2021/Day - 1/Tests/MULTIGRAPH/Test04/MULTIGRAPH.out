559696224
2195884530
23865199071
44662154176
36082398312
53132135116
40065553308
27972017854
50538823570
99601726150
81105733275
137720279088
180989952234
175022146708
126255276882
264602762650
238593487686
170244704152
198307578978
145807173894
88425186298
135051103146
138721126152
90771323458
309317519848
237457961544
342519625632
306825187490
450122574209
271485639648
429653870064
159552395136
507063383750
486673375834
550059863796
510443165841
272051466912
381479437010
592561716365
626673896220
383575392504
490239818438
551985014646
353795929508
644258707372
429729063972
389157161280
441813761910
472031482440
493915470468
595443941232
621404198184
